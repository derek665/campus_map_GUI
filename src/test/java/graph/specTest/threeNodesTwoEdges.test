# Tests that a graph can be created, three nodes and
# two edges from the same node, and the three nodes
# and the two edges can be printed out.

# Create a graph
CreateGraph g1

# Add 3 nodes to graph
AddNode g3 n1
AddNode g3 n2
AddNode g3 n3

# Add 2 edges from n1
AddEdge g3 n1 n2 e1
AddEdge g3 n1 n3 e2

# Print nodes in graph and edges from n1
ListNodes g3
ListChildren g3 n1
