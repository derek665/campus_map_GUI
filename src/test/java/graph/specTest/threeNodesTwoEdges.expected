# Tests that a graph can be created, three nodes and
# two edges from the same node, and the three nodes
# and the two edges can be printed out.

# Create